tests/UI/campaigns/functional/BO/11_international/01_localization/currencies/05_filterSortAndPagination.js

```diff
@@ -0,0 +1,304 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+
+// Import pages
+const dashboardPage = require('@pages/BO/dashboard');
+const localizationPage = require('@pages/BO/international/localization');
+const currenciesPage = require('@pages/BO/international/currencies');
+const addCurrencyPage = require('@pages/BO/international/currencies/add');
+
+// Import Data
+const {Currencies} = require('@data/demo/currencies');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_international_localization_currencies_filterSortAndPagination';
+
+let browserContext;
+let page;
+let numberOfCurrencies = 0;
+
+/*
+Create 10 official currencies
+Filter currencies by: ID, currency, symbol, iso code, enabled
+Pagination next and previous
+Sort currencies by : ID, iso code, exchange rate, enabled
+Delete the created currencies
+ */
+describe('Create official currency and check it in FO', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  it('should login in BO', async function () {
+    await loginCommon.loginBO(this, page);
+  });
+
+  it('should go to localization page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToLocalizationPage', baseContext);
+
+    await dashboardPage.goToSubMenu(
+      page,
+      dashboardPage.internationalParentLink,
+      dashboardPage.localizationLink,
+    );
+
+    await localizationPage.closeSfToolBar(page);
+
+    const pageTitle = await localizationPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(localizationPage.pageTitle);
+  });
+
+  it('should go to currencies page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCurrenciesPage', baseContext);
+
+    await localizationPage.goToSubTabCurrencies(page);
+
+    const pageTitle = await currenciesPage.getPageTitle(page);
+    await expect(pageTitle).to.contains(currenciesPage.pageTitle);
+  });
+
+  it('should reset all filters', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'resetFilterFirst', baseContext);
+
+    numberOfCurrencies = await currenciesPage.resetAndGetNumberOfLines(page);
+    await expect(numberOfCurrencies).to.be.above(0);
+  });
+
+  const currencies = [Currencies.mad, Currencies.all, Currencies.chileanPeso, Currencies.dzd, Currencies.tnd,
+    Currencies.try, Currencies.usd, Currencies.aed, Currencies.lyd, Currencies.lsl,
+  ];
```

Comment:
To Avoid, creating all this currencies, you can try creating unofficial currencies (you don't have to add them in demo file too).